pkg/image/importer/importer.go

```diff
@@ -50,6 +50,9 @@ type ImageStreamImporter struct {
 	limiter   flowcontrol.RateLimiter
 
 	digestToRepositoryCache map[gocontext.Context]map[manifestKey]*api.Image
+
+	// digestToLayerSizeCache maps layer digests to size.
+	digestToLayerSizeCache map[string]int64
```

Comment:
Can we make it thread-safe. And let it be passed to `NewImageStreamImporter` so it can be shared among multiple importers invoked by image import controller?